xcube/util/tilegrid.py

```diff
@@ -24,11 +24,14 @@
 from typing import Tuple, Optional, Union, Dict, Any
 
 import pyproj
+from deprecated import deprecated
 
 from xcube.util.assertions import assert_instance
 from xcube.util.assertions import assert_true
 
 
+@deprecated(version='0.11.0',
+            reason='use xcube.util.tilegrid2.TileGrid2 instead')
```

Comment:
Please do not reference classes that do not exist (yet). This also holds for the other occrurrences of this deprecation message in this module.

Reply:
The class referenced here should be the new `xcube.core.tilingscheme.TilingScheme`. Will fix.